.vscode/cspell.global.yaml

```diff
@@ -121,6 +121,7 @@ ignoreWords:
     - aztfmod
```

Comment:
Please have a look at this update to bicep and try to follow the same for tf.  Thanks https://github.com/Azure/azure-dev/pull/760

Reply:
Thanks @jongio checked it out. I do not pass app insights values as input parameters to any of the terraform modules.